eng/build.ps1

```diff
@@ -264,7 +264,7 @@ function TestUsingOptimizedRunner() {
     $dlls = $dlls | ?{ -not ($_.FullName -match ".*/ref/.*") }
 
     if ($ci) {
-        $args += " -xml -timeout:65"
+        $args += " -xml -timeout:120"
```

Comment:
We should condition this timeout difference on the `$testVsi` variable. Don't want to move the unit tests to 120 because it lets people slip in bad changes.

Reply:
➡️ Fixed. Now the timeout increase only occurs when `$testVsi` is true.